smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/protocols/AwsRestJson1ProtocolTest.java

```diff
@@ -94,4 +94,29 @@ public void canRemoveGreedyLabelNameParameterSuffix() {
             Node.assertEquals(result, expectedNode);
         }
     }
+
+    @Test
+    public void canRemoveNonAlphaNumericDocumentNames() {
+        String smithy = "non-alphanumeric-content-names.json";
+        Model model = Model.assembler()
+                .addImport(getClass().getResource(smithy))
+                .discoverModels()
+                .assemble()
+                .unwrap();
+        OpenApiConfig config = new OpenApiConfig();
+        config.setService(ShapeId.from("smithy.example#Service"));
+        config.setAlphanumericOnlyRefs(true);
+        OpenApi result = OpenApiConverter.create()
+                .config(config)
+                .convert(model);
+        String openApiModel = smithy.replace(".json", ".openapi.json");
+        InputStream openApiStream = getClass().getResourceAsStream(openApiModel);
+
+        if (openApiStream == null) {
+            LOGGER.warning("OpenAPI model not found for test case: " + openApiModel);
```

Comment:
you should fail the test here

Reply:
I updated all the cases to fail in 5333705.